packages/react-dom/src/client/ReactDOM.js

```diff
@@ -786,12 +786,7 @@ ReactRoot.prototype.unmount = function(callback) {
   DOMRenderer.updateContainer(null, root, null, callback);
 };
 
-var ReactDOM = {
-  createRoot(container: DOMContainer, options?: RootOptions): ReactRootNode {
-    const hydrate = options != null && options.hydrate === true;
-    return new ReactRoot(container, hydrate);
-  },
-
+var ReactDOM: Object = {
```

Comment:
For my own understanding of Flow, does `: Object` distinguish `ReactDOM`'s type over what Flow can infer?

Reply:
I guess this relaxes the type so we can add things to it later. Actually this is probably not a great pattern and we should stop doing this. Since it probably also relaxes which properties it lets us read.

Reply:
We should make sure to remove the `: Object` when removing the `enableCreateRoot` flag.